server/services/offendersService.js

```diff
@@ -455,6 +455,38 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
     }
   }
 
+  async function getPrisonerBackground(token, offenderNo) {
+    try {
+      const nomisClient = nomisClientBuilder(token)
+      const currentCats = await nomisClient.getCategoryHistory(offenderNo)
+
+      const decoratedCats = await Promise.all(
+        currentCats.map(async o => {
+          const description = await getOptionalAssessmentAgencyDescription(nomisClient, o.assessmentAgencyId)
+          const assessmentMoment = moment(o.assessmentDate, 'YYYY-MM-DD')
```

Comment:
dateConverter() will sort this , its available in the html too

Reply:
there are a few dotted around - I'll raise a minor tech debt ticket for this